client-v2/src/components/FrontsEdit/ArticleFragmentFormInline.tsx

```diff
@@ -435,23 +439,15 @@ class FormComponent extends React.Component<Props, FormComponentState> {
                       : articleCapiFieldValues.thumbnail
                   }
                   useDefault={!imageCutoutReplace && !imageReplace}
-                  message={imageCutoutReplace ? 'Add cutout' : 'Add image'}
+                  message={imageCutoutReplace ? 'Add cutout' : 'Replace image'}
                   onChange={this.handleImageChange}
+                  setDisplayImageReplaceToggle={
+                    this.setDisplayImageReplaceToggle
+                  }
+                  setImageReplaceToggleValue={this.setImageReplaceToggleValue}
```

Comment:
See below comment re: these new fields -- could we do this in `onChange`, a bit like line 498? (God this form is long)